src/archive.rs

```diff
@@ -36,4 +39,57 @@ pub struct Header {
     pub terminator: [u8; 2],
 }
 
+/// The header at the start of an AIX big archive member, without name.
+#[derive(Debug, Clone, Copy)]
+#[repr(C)]
+pub struct AixHeader {
+    /// Member size in decimal.
+    pub size: [u8; 20],
+    /// Offset of next member in decimal.
+    pub next_member: [u8; 20],
+    /// Offset of previous member in decimal.
+    pub prev_member: [u8; 20],
+    /// File modification timestamp in decimal.
+    pub date: [u8; 12],
+    /// User ID in decimal.
+    pub uid: [u8; 12],
+    /// Group ID in decimal.
+    pub gid: [u8; 12],
+    /// File mode in octal.
+    pub mode: [u8; 12],
+    /// Name length in decimal.
+    pub name_length: [u8; 4],
+}
+
+/// Discriminated union for multiple type headers
+#[derive(Debug, Clone, Copy)]
+pub enum MemberHeader {
+    /// GNU or BSD style header
+    SystemV(Header),
+    /// AIX style big archive header
+    AixBig(AixHeader),
+}
```

Comment:
Maybe use `&'a Header` and `&'a AixHeader` to avoid copies.

Reply:
Yes this can use references. Also move this to `src/read/archive.rs` and don't make it public.